test/util/regexParser.js

```diff
@@ -1,403 +1,351 @@
-regexParser = (function() {
```

Comment:
I had to convert this file to a valid ES module. I've created #7 for integrate PEG.js in the build process.